src/common/test/TestCommon.cpp

```diff
@@ -303,6 +303,34 @@ TEST(CommonTest, TestTimeParsing) {
   }
 }
 
+TEST(CommonTest, TestStripTime) {
```

Comment:
The current tests are good, do you have more tests (especially edge cases) to add? These core functions deserve more coverage.